home.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+  <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+  <link rel="stylesheet" href="home.css">
+  
+  <title>Home-page</title>
+</head>
+<body >
+  <header>
+    <nav class="nav-top">
+        <ul class="">
+       
+         <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+    <li class="top"><a href="#">English</a></li>
+    <li><a href="#">My page</a></li>
+    <li><a href="#">Logout</a></li>
+        
+        </ul>
```

Comment:
- [ ] On your mobile version, kindly try as much as you can to replicate the original design. Most importantly, you can consider spacing your elements correctly. This includes using the correct font and providing margins and paddings correctly. I encountered an issue with the mobile menu. Unfortunately, it is not functioning properly, which can be frustrating for users trying to navigate the website on their mobile devices. The menu is a crucial component for easy navigation and accessibility, so it would be beneficial to address this issue as soon as possible.

![image](https://github.com/mumbereh/Capston-project/assets/98453979/6ac69e2e-b8bf-4d2e-8e6c-63feaef2d52c)

![image](https://github.com/mumbereh/Capston-project/assets/98453979/9500fd80-bcdb-4b40-912a-a10cb5a44140)

![image](https://github.com/mumbereh/Capston-project/assets/98453979/a83b2f66-e647-4577-8b21-d1c0a881c063)